functions/helloworld/lib/functions.dart

```diff
@@ -0,0 +1,9 @@
+// [START functions_helloworld_get]
```

Comment:
What are these `[START` and `[END` markers? Is that markup used for publishing the examples to docs? Any other markup or naming conventions I should be aware of?